src/lxc/lxccontainer.c

```diff
@@ -1320,7 +1322,7 @@ static bool create_run_template(struct lxc_container *c, char *tpath, bool need_
 		}
 		/* execute */
 		execvp(tpath, newargv);
-		SYSERROR("failed to execute template %s", tpath);
+		SYSERROR("Failed to execute template %s", tpath);
 		exit(1);
```

Comment:
@0x0916, good that you ask that. This is something we came to agree on only while back. I think the idea is to have the first letter be uppercase (if appropriate; there might be exceptions) and not end with a dot.